addons/mail/tests/test_mail_composer.py

```diff
@@ -137,27 +137,6 @@ def test_composer_default_recipients_private(self):
         self.assertEqual(message.partner_ids, partner_private + partner_classic)
         self.assertEqual(message.subject, f'{test_record.name}')
 
-    @mute_logger('odoo.addons.base.models.ir_rule', 'odoo.addons.mail.models.mail_mail')
-    @users('employee')
-    def test_composer_default_recipients_private_norights(self):
```

Comment:
Do not remove tests, especially those linked to ACLs on partners. If you decide to functionally change the 17.0 (which is something I tend to disagree), you have to highlight the new behavior by showing that 1/ we can't access private partners and 2/ what happens when using them. Removing the test is just removing work done previously, notably at odoo/odoo#107642 (which was a pita as far as I remember).

Reply:
This test is flawed because you can access it by just setting the context key `active_test=False`...

Reply:
Wasn't the test broken when the "private" partners were removed ?

Reply:
No, it still raises, but only because of an implementation detail based on the context, which is not good for a security feature